NewConnect.php

```diff
@@ -9,82 +9,241 @@
+                            <?php  if (isset($_SESSION['id'])) : ?>
+                              <a id="notif_trigger" href="#" onclick = notify_alert(<?php echo $_SESSION[ 'id'] ?>) style = "text-decoration: none;"><i class = "fa fa-bell"></i>
+                &nbsp;&nbsp;<span id="notif_count" style="color:white" ><?php echo $notif_count ?></span></a> &nbsp;&nbsp;
+                                <a class="google" href='  welcome.php' style="text-decoration: none;">&nbsp;&nbsp;Back to Dashboard&nbsp;&nbsp;</a>&nbsp;&nbsp;&nbsp;&nbsp;
 
-<a class = "google" href = 'http://localhost/connect/welcome.php?logout=1'>
-        &nbsp;&nbsp;Logout&nbsp;&nbsp;
-      </a>&nbsp;&nbsp;&nbsp;&nbsp;
-        <a class = "google" href = 'http://localhost/connect/finalsignup.php'>&nbsp;Signup&nbsp;&nbsp;</a>
-      <?php endif ?>
-      </object>  
-</object>
-</div>   
-<br>
-<div class="connect">
-<center> <div class = "signup_box info" style = "height: 400px; width: 90%;"><br>
-<table>
-<tr>
-<th>Id</th>
-<th>Name</th>
-<th>Username</th>
-
-<th>Bio</th>
-
-</tr>
-<?php
-$conn = mysqli_connect("localhost", "root", "abiit@2019", "rconnect");
-// Check connection
+                                <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+                                            More Options..
+                                </button>
+                                        <div class="dropdown-menu" style="z-index: 2;">
+                                            <a class="dropdown-item google" href='welcome.php?logout=1'>Logout</a>
+                                            <a class="dropdown-item google" href='welcome.php?delete=1'>Delete my Account</a>
+                                        </div>
+                                <?php endif ?>
+                </object>
+            </object>
+        </div>
+        <br>
+        <div class="connect">
+            <center>
+                <div class="signup_box info" style="height: 600px; width: 100%; overflow:auto;">
+                    <br>
+                    <table class="table table-dark table-hover" id="user-details">
+                        
+                     
+                    </table>
+                </div>
+        </div>
 
-$sql = "SELECT userID, name, username, password, bio FROM userData";
+        <div id="user_model_details" style="width: 400px"></div>        
+        </center>
+<?php
+include('dbconfig.php');
+$sql = "SELECT userID, name, email, bhawan, username, img, bio, branch_y, follow_count FROM userData ORDER BY follow_count DESC";
 $result = mysqli_query($conn, $sql);
+function hobbieID_to_name($a){
+  if($a==1){return "Dancing";}
+  if($a==2){return "Listening Music";}
+  if($a==3){return "Cricket";}
+  if($a==4){return "Singing";}
+  if($a==5){return "Fooseball";}
+  if($a==6){return "Reading Novels";}
```

Comment:
why is this hardcoded here , can you find a better way to do that

Reply:
Yeah, imo using arrays for the task would be better. I'll just update the code

Reply:
Changed the code to array format